Plugins/TGeo/src/TGeoSurfaceConverter.cpp

```diff
@@ -93,16 +93,21 @@ Acts::TGeoSurfaceConverter::cylinderComponents(const TGeoShape& tgShape,
       // Check if it's a segment
       auto tubeSeg = dynamic_cast<const TGeoTubeSeg*>(tube);
       if (tubeSeg != nullptr) {
-        double phi1 = toRadian(tubeSeg->GetPhi1());
-        double phi2 = toRadian(tubeSeg->GetPhi2());
-        if (not boost::starts_with(axes, "X")) {
-          throw std::invalid_argument(
-              "TGeoShape -> CylinderSurface (sectorial): can only be converted "
-              "with "
-              "'(X)(y/Y)(*)' axes.");
+        double gradPhi1 = tubeSeg->GetPhi1();
+        double gradPhi2 = tubeSeg->GetPhi2();
+        if (not(gradPhi1 == 0. and gradPhi2 == 360.)) {
```

Comment:
```suggestion
        if (not(gradPhi1 <= 0. and gradPhi2 >= 360.)) {
```
1. Can we trust root on the limits?

2. Should we also add this check in l.249? Maybe even have a helper function for this?

Reply:
Can we trust ROOT on ANYTYHING ? 

I think the check is good as is, because what we want to catch is that someone did not set it, in case the defaults are taken, which are checked against. I will check if there's a better way to see if the shape is closed.

Reply:
Ok, I moved to a check on `s_epsilon`.